starcheck/calc_ccd_temps.py

```diff
@@ -234,7 +234,10 @@ def get_ccd_temps(oflsdir, outdir='out',
     make_check_plots(outdir, states, ccd_times, ccd_temps,
                      tstart=bs_start.secs, tstop=sched_stop.secs)
     intervals = get_obs_intervals(sc_obsids)
-    obsreqs = None if orlist is None else {obs['obsid']: obs for obs in read_or_list(orlist)}
```

Comment:
Not an actual review but you can keep the ternary with 
```
obsreqs = None if orlist is None else read_or_list_full(orlist)[0]
```

Reply:
👍  Though with regard to only taking or caring about some of the returned tuple of a function -- which is preferrred?  Indexing into the tuple or explicitly discarding with `_`?  I'm fine with either.

Reply:
It depends. In general prefer discarding with `_`, but in this case keeping the ternary structure using indexing works better.